### DISCUSSION TITLE
Unable to install Searchkit

### DISCUSSION THREAD

**Discussion by ravibyte** [searchkit/searchkit#1306] [Q&A]
Searchkit team,
When I try to install SearchKit via npm install, I am getting following errors. There are lot of conflicts between eslint,prettier and eslint-plugin-prettier versions. I tried too many times so far in different environments and results are similar to below errors.

While resolving: @typescript-eslint/eslint-plugin@4.33.0
npm ERR! Found: eslint@8.48.0
npm ERR! node_modules/eslint
npm ERR!   peer eslint@"^6.0.0 || ^7.0.0 || >=8.0.0" from @eslint-community/eslint-utils@4.4.0
npm ERR!   node_modules/@eslint-community/eslint-utils
npm ERR!     @eslint-community/eslint-utils@"^4.2.0" from eslint@8.48.0
npm ERR!   peer eslint@"*" from @typescript-eslint/experimental-utils@4.33.0
npm ERR!   node_modules/@typescript-eslint/experimental-utils
npm ERR!     @typescript-eslint/experimental-utils@"4.33.0" from @typescript-eslint/eslint-plugin@4.33.0
npm ERR!     node_modules/@typescript-eslint/eslint-plugin
npm ERR!       dev @typescript-eslint/eslint-plugin@"^4.4.1" from the root project
npm ERR!   12 more (eslint-utils, eslint-config-next, ...)

**Comment by ravibyte** [comment]
If I run "npm install searchkit @searchkit/api @searchkit/instantsearch-client", I am getting following errors

ode ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: @typescript-eslint/eslint-plugin@4.33.0
npm ERR! Found: eslint@8.48.0
npm ERR! node_modules/eslint
npm ERR!   peer eslint@"^6.0.0 || ^7.0.0 || >=8.0.0" from @eslint-community/eslint-utils@4.4.0
npm ERR!   node_modules/@eslint-community/eslint-utils
npm ERR!     @eslint-community/eslint-utils@"^4.2.0" from eslint@8.48.0
npm ERR!   peer eslint@"*" from @typescript-eslint/experimental-utils@4.33.0
npm ERR!   node_modules/@typescript-eslint/experimental-utils
npm ERR!     @typescript-eslint/experimental-utils@"4.33.0" from @typescript-eslint/eslint-plugin@4.33.0
npm ERR!     node_modules/@typescript-eslint/eslint-plugin
npm ERR!       dev @typescript-eslint/eslint-plugin@"^4.4.1" from the root project
npm ERR!   12 more (eslint-utils, eslint-config-next, ...)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer eslint@"^5.0.0 || ^6.0.0 || ^7.0.0" from @typescript-eslint/eslint-plugin@4.33.0
npm ERR! node_modules/@typescript-eslint/eslint-plugin
npm ERR!   dev @typescript-eslint/eslint-plugin@"^4.4.1" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: eslint@7.32.0
npm ERR! node_modules/eslint
npm ERR!   peer eslint@"^5.0.0 || ^6.0.0 || ^7.0.0" from @typescript-eslint/eslint-plugin@4.33.0
npm ERR!   node_modules/@typescript-eslint/eslint-plugin
npm ERR!     dev @typescript-eslint/eslint-plugin@"^4.4.1" from the root project

**Comment by joemcelroy** [comment]
Could you share me your contents of package.json? Will investigate.

I checked the following:
- Works as expected on a brand new project (via https://nextjs.org/learn/basics/create-nextjs-app/setup) and installed without issues
. Checked to make sure eslint didn't sneak into searchkit dependencies

I suspect your error could be unrelated to searchkit. Try with another package and see if you encounter npm issues. Worth deleting and reinstalling node_modules  and/or running in vebose mode to confirm if its searchkit thats conflicting with eslint.

**Comment by ravibyte** [comment]
Thanks @joemcelroy Let me try few more times and will get back to you.
